code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -484,6 +487,11 @@ namespace <#= qualifiedNamespace #>
                 ComponentType.ReadOnly<<#= qualifiedNamespace #>.<#= componentDetails.ComponentName #>.CommandResponders.<#= commandDetails.CommandName #>>(),
 <# } #>
             };
+            
+            public override ComponentType[] AuthorityLossComponentTypes => new ComponentType[] {
+                ComponentType.ReadOnly<AuthorityLossImminent<<#= componentDetails.TypeName #>>>(),
```

Comment:
This shouldn't be read only as you modify the data

Reply:
Would the correct thing be to replace it by `typeof(AuthorityLossImminent<SpatialOSTransform>)`?

Reply:
`ComponentType.Create<AuthorityLossImminent<...>>`